examples/example.go

```diff
@@ -8,7 +8,7 @@ import (
 
 func main() {
 	patch := godiffpatch.GeneratePatch("test.txt", original, updated)
-	err := os.WriteFile("gitignore.patch", []byte(patch), 0644)
+	err := os.WriteFile("test.txt", []byte(patch), 0644)
```

Comment:
Is it any prettier just to print the difference or is a file easier to read?

Reply:
It was easier for me to read from a file when experimenting initially so the logic stayed. I'm not opposed to printing it though.